### ISSUE TITLE
Change absolute links to relative links 

### ISSUE THREAD

**Issue by annalorimer** [wics-uw/website#222]
There have been discussions about changing all links that link to a page on wics.uwaterloo.ca to be relative instead of absolute.

Discuss below!

Here's some research:

http://www.navegabem.com/linking-strategy.html
https://moz.com/blog/relative-vs-absolute-urls-whiteboard-friday
http://stackoverflow.com/questions/2005079/absolute-vs-relative-urls

**Comment by ehashman** [comment]
Here's the git grep (minus the ones that don't need to be changed):

```
$ git grep 'wics.uwaterloo.ca'
content/F2015/F15-blog-wics-sys.md:2. [Join our IRC channel.](http://wics.uwaterloo.ca/irc/) IRC, [while
content/F2015/F15-event-orientation.md:Note: This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
content/F2015/F15-event-orientation.md:+ **Who:** Incoming [Women](http://wics.uwaterloo.ca/faq/) in CS
content/S2015/S15-event-wicsstudies.md:[women-only](http://wics.uwaterloo.ca/faq/) event.
content/S2016/S16-event-boardgames.md:Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
content/S2016/S16-event-boardgames.md:+ **Who:** [Women and non-binary individuals](http://wics.uwaterloo.ca/faq).
content/S2016/S16-event-csters-destress.md:Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
content/S2016/S16-event-csters-destress.md:+ **Who:** [Women and non-binary individuals](http://wics.uwaterloo.ca/faq).
content/S2016/S16-event-square.md:Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
content/S2016/S16-event-square.md:+ **Who:** [Cis & trans women and non-binary individuals](http://wics.uwaterloo.ca/faq)
content/S2016/S16-event-teaparty.md:a Big CSters event is and is thus [women-only](http://wics.uwaterloo.ca/faq).
content/S2016/S16-event-teaparty.md:           [women and non-binary individuals](http://wics.uwaterloo.ca/faq)
content/W2016/W16-event-airbnb.md:Please note that this is a [women-only](http://wics.uwaterloo.ca/faq) event.
content/W2016/W16-event-gitworkshop.md:Note: This is a [women-only](http://wics.uwaterloo.ca/faq) event.
content/W2016/W16-event-gitworkshop.md:+ **Who:** [Women-only](http://wics.uwaterloo.ca/faq)
content/W2016/W16-event-teaparty.md:a Big CSters event is and is thus [women-only](http://wics.uwaterloo.ca/faq).
content/pages/sponsors.md:[here](http://wics.uwaterloo.ca/tags/blackberry). In Fall 2015 we ran our 
```

It appears that the FAQ absolute link got cargo-culted in S15 and has been showing up since then. All these files from the git grep should get updated.

**Comment by bdamerac** [comment]
Almost done. What's the relative link for 'http://wics.uwaterloo.ca/tags/blackberry' ?

**Comment by ehashman** [comment]
It should be `{tag}blackberry` or the like.

**Comment by evykassirer** [comment]
fixed in #230
